blocks/cards/cards.js

```diff
@@ -43,6 +44,12 @@ export default async function decorate(block) {
         const bubble = div.querySelector('u');
         if (bubble) {
           bubble.className = 'cards-card-bubble';
+          bubble.closest('p').className = 'cards-card-bubble-wrapper';
+        }
+        const country = div.querySelector('.icon[class*=icon-flag-]');
+        if (country) {
+          country.closest('p').classList.add('cards-card-country');
+          div.classList.add('cards-card-country-wrapper');
```

Comment:
leveraging and mirroring existing bubble logic (bubble is not unique to "champion" variant)